plugin.video.nfl.gamepass/addon.xml

```diff
@@ -5,7 +5,7 @@
      marooned on an old version. -->
 <addon id="plugin.video.nfl.gamepass"
        name="NFL Game Pass"
-       version="0.11.1"
+       version="2018.08.15-release.1"
```

Comment:
I am not sure this is valid. @rechi can u help?

Reply:
@enen92 Ahh oops. I should have checked first. Let me know if it needs to be changed.

Reply:
it might simply just work however it's not according to our x.y.z standard we try to keep

Reply:
@MartijnKaijser Thanks for the clarification. I've pushed a version with this fixed.
@enen92 I've squashed the fix and pushed it. It should be good now.